_includes/contributor/2-1-end.md

```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-warning}
+**v2.1 End of Contribution:**
+As of September 15, 2018, we no longer accept contributions to the v2.1 release line. Please submit contributions for the [v2.2](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html) and [v2.3](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html) release lines. We made this change to increase focus on issue investigations and pull request approvals for v2.2 and v2.3 release lines.
```

Comment:
I would take "Please" out. Something like "Continue submitting contributions for the ..."

"We made this change to increase our focus.."

Reply:
Updated!